src/components/APIDoc/SchemaDataView.tsx

```diff
@@ -128,7 +128,13 @@ const getTreeViewData = (schemaName: string, schema: DeRefResponse<OpenAPIV3.Arr
   if (!schema.properties) {
     return [{name: "schema undefined"}] as TreeViewDataItem[]
   }
-  const schemaData = Object.entries(schema.properties).map(([key, value]) => {
+
+  const schemaKeyValArray = Object.entries(schema.properties)
+  if (schemaKeyValArray.length < 1) {
+    return [{name: "schema undefined"}] as TreeViewDataItem[]
```

Comment:
We need a better name,  `properties: {}` means "You can write anything in here" and could be used when you want to receive arbitrary data. Perhaps `Any object` or `Any data` ?

Reply:
Ok, changing to `Any data` :+1: